src/test/java/org/prebid/server/bidder/bizzclick/BizzclickBidderTest.java

```diff
@@ -0,0 +1,401 @@
+package org.prebid.server.bidder.bizzclick;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Device;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Native;
+import com.iab.openrtb.request.Video;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.vertx.core.MultiMap;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.bizzclick.ExtImpBizzclick;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static java.util.function.UnaryOperator.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.groups.Tuple.tuple;
+
+public class BizzclickBidderTest extends VertxTest {
+
+    private static final String URL_SOURCE_ID_MACRO = "{{.SourceId}}";
+    private static final String URL_ACCOUNT_ID_MACRO = "{{.AccountID}}";
+    private static final String URL_TEMPLATE = "https://test.domain.dm/uri?source=%s&account=%s";
+
+    private BizzclickBidder bidder;
+
+    @Before
+    public void setUp() {
+        final String url = String.format(URL_TEMPLATE, URL_SOURCE_ID_MACRO, URL_ACCOUNT_ID_MACRO);
+        bidder = new BizzclickBidder(url, jacksonMapper);
+    }
+
+    @Test
+    public void creationShouldFailOnInvalidEndpointUrl() {
+        assertThatIllegalArgumentException().isThrownBy(() -> new BizzclickBidder("incorrect_url", jacksonMapper));
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenImpExtCouldNotBeParsed() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                givenImp(imp -> imp.ext(mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode())))));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = bidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).extracting(BidderError::getMessage)
```

Comment:
Very minor. Please, place extracting calls on the next line, check for similar occurences.